src/components/User/Admins/components/AdminManager.vue

```diff
@@ -250,20 +261,22 @@ export default {
           if (+item.id === +MASTER_ROLE_ID) {
             return {
               id: item.id,
-              name: 'Master',
-              description: 'The root admin of the system',
+              name: globalize('admin-manager.master'),
+              description: globalize('admin-manager.root-admin-system'),
             }
           }
 
           return {
             id: item.id,
-            name: (item.details || {}).name || `Unnamed (${item.id})`,
-            description: (item.details || {}).description || '(No description)',
+            name: (item.details || {}).name ||
+            globalize('admin-manager.unnamed')`(${item.id})`,
```

Comment:
Are you sure this works correctly?
If 'no', why this code stay here

Reply:
done